packages/network/src/composition.ts

```diff
@@ -102,7 +103,8 @@ export const createNetworkNode = ({
     newWebrtcConnectionTimeout,
     webrtcDatachannelBufferThresholdLow,
     webrtcDatachannelBufferThresholdHigh,
-    stunUrls = ['stun:stun.l.google.com:19302']
+    stunUrls = ['stun:stun.l.google.com:19302'],
+    trackerConnectionMaintenanceInterval
```

Comment:
Added this to constructor (as opposed to hard-coded value) as it is useful to be able to make the interval very small in tests in furtherance of fast test times.